### PR TITLE
[Docs] Add a single-point starter guide for contributors 

### PR DESCRIPTION
**Pull request by p-wysocki**
### Details:
 - Add a single point of entry into OpenVINO documentation
 - The document guides new OpenVINO contributors from cloning the repository all the way to submitting a Pull Request
 - The links allow for quick access to all (?) important documentation starting points for each component

### TODO
- ~~I believe we should simplify CONTRIBUTING.md, CONTRIBUTING_ROADMAP.md, get_started.md and so on, preferably merge into one~~
- ~~"Roadmap" may be a confusing name, this document may require name change~~

### Tickets:
 - N/A

### CODE REVIEW

File: CONTRIBUTING_ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- [PaddlePaddle Frontend](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/paddle)
+- [PyTorch Frontend](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/pytorch)
+- [TensorFlow Frontend](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/tensorflow)
+
+#### Plugins
+- [Auto plugin](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto)
+- [CPU plugin](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu)
+- [GPU plugin](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu)
+- [Hetero plugin](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/hetero)
+- [Template plugin](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/README.md)
```
Comment (p-wysocki) [inline CONTRIBUTING_ROADMAP.md:41]: TODO: Fix link

**Review by mitruska — COMMENTED** [review]
As we discussed, it's worth to consider a FAQ (Frequently Asked Questions) section. 
It could be supported by links to the already existing useful comments from reviewers/maintainers for issues, where such question was already resolved.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -81,6 +87,66 @@ product better:
      share your expertise with the community. Check GitHub Discussions and 
      Issues to see if you can help someone. 
 
+## Technical guide
+
+This section lists all the necessary steps required to setup your environment, build OpenVINO locally and run tests for specific components. It's a perfect place to start when you have just picked a Good First Issue and are wondering how to start working on it.
```
Comment (kblaszczak-intel) [inline CONTRIBUTING.md:92]:
```suggestion
This section lists all the necessary steps required to set up your environment, build OpenVINO locally, and run tests for specific components. It's a perfect place to start when you have just picked a Good First Issue and are wondering how to start working on it.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -81,6 +87,66 @@ product better:
...
+
+Keep in mind that we are here to help - **do not hesitate to ask the development team if something is not clear**. Such questions allow up to keep improving our documentation.
```
Comment (kblaszczak-intel) [inline CONTRIBUTING.md:94]:
```suggestion
Keep in mind that we are here to help - **do not hesitate to ask the development team if something is not clear**. Such questions allow us to keep improving our documentation.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -81,6 +87,66 @@ product better:
...
+You can start with the following links:
+- [What is OpenVINO?](https://github.com/openvinotoolkit/openvino#what-is-openvino-toolkit)
+- [OpenVINO architecture](https://github.com/openvinotoolkit/openvino/blob/master/src/docs/architecture.md)
+- [User documentation](https://docs.openvino.ai/)
+- [Blog post on contributing to OpenVINO](https://medium.com/openvino-toolkit/how-to-contribute-to-an-ai-open-source-project-c741f48e009e)
+- [Pick up a Good First Issue](https://github.com/orgs/openvinotoolkit/projects/3)
+
+### 2. Building the project
+
+In order to build the project, please follow the [build instructions for your specific OS](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md).
```
Comment (kblaszczak-intel) [inline CONTRIBUTING.md:107]:
```suggestion
In order to build the project, follow the [build instructions for your specific OS](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md).
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -81,6 +87,66 @@ product better:
...
+
+### 3. Familiarize yourself with component you'll be working with
```
Comment (kblaszczak-intel) [inline CONTRIBUTING.md:109]:
```suggestion
### 3. Familiarize yourself with the component you'll be working with
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -81,6 +87,66 @@ product better:
...
+
+Choose the component your Good First Issue is related to. You can run tests to make sure it is working correctly.
```
Comment (kblaszczak-intel) [inline CONTRIBUTING.md:111]:
```suggestion
Choose the component your Good First Issue is related to. You can run tests to make sure it works correctly.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -81,6 +87,66 @@ product better:
...
+
+### 3. Start working on your Good First Issue
+
+Use the issue description and locally built OpenVINO to complete the task. Remember that you can always ask users tagged in the "Contact points" section for help!
+
+### 4. Submit a PR with your changes
+
+Follow our [Good Pull Request guidelines](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING_PR.md).
+
+### 5. Wait for review
```
Comment (kblaszczak-intel) [inline CONTRIBUTING.md:147]:
```suggestion
### 5. Wait for a review
```

**Review by kblaszczak-intel — APPROVED** [review]
That is a very cool addition! I've added tiny language adjustments but they are barely relevant.

**Comment by p-wysocki** [comment]
@jiwaszki @mitruska I'll address your suggestions in a separate PR, I really want to merge this one ASAP. Just letting you know that I did not forget about them. :)

**Comment by mlukasze** [comment]
please add ticket ID which you will use to follow up

**Pull request merged**